### ISSUE TITLE
pl-drawing provides no way to create accessible diagrams

### ISSUE THREAD

**Issue by reteps** [PrairieLearn/PrairieLearn#11841]
### Details

Currently, all our `pl-drawing` examples have inaccessible diagrams since no alt is given. Perhaps alt text can be generated based on the items added to the canvas?

If no alt makes sense for these diagrams, perhaps providing a generic `alt` text for these diagrams would be sufficient.

**Comment by jonatanschroeder** [comment]
Missing alt is one concern for static diagrams, but interactivity in this element is a bigger concern. Afaik there are no viable keyboard controls or reasonable screen reader interaction.

**Comment by jonatanschroeder** [comment]
See #7384.

**Comment by reteps** [comment]
@jonatanschroeder In the hopes of improving support for static diagrams, I'm going to make a PR for alt text, with a caveat for interactive diagrams on the documentation.

**Comment by reteps** [comment]
With the addition of #11867 and ways to create optional fallback content through #11929 , this issue in my opinion should be closed.
